src/convertToObject.js

```diff
@@ -5,8 +5,106 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const newObject = {};
+
+  const splitStrings = sourceString.split(';');
+
+  const formateStrings = splitStrings.map((param) => {
+    const newString = param
+      .replaceAll('\n', '')
+      .replaceAll(',', `,\n`)
```

Comment:
Similar to the previous comment, use `.replace(/,/g, ',\n')` instead of `replaceAll` for better compatibility.